src/data/injections.js

```diff
@@ -159,23 +159,6 @@ const AVAILABLE_INJECTIONS = [
     },
     customFunc: "pdk5fix",
   },
-  {
-    id: "bug1577870",
-    platform: "desktop",
-    domain: "Download prompt for files with no content-type",
-    bug: "1577870",
-    data: {
-      urls: [
-        "https://ads-us.rd.linksynergy.com/as.php*",
-        "https://www.office.com/logout?sid*",
-      ],
-      contentType: {
-        name: "content-type",
-        value: "text/html; charset=utf-8",
-      },
-    },
-    customFunc: "noSniffFix",
```

Comment:
We also need to remove the custom function (noSniffFix and noSniffFixDisable):
https://github.com/mozilla-extensions/webcompat-addon/blob/master/src/lib/custom_functions.js#L58-L74

Reply:
@ksy36  Thanks.

https://github.com/mozilla-extensions/webcompat-addon/blob/master/src/lib/custom_functions.js#L58-L74

What about 

https://github.com/mozilla-extensions/webcompat-addon/blob/72e35293a0e4f4a0c34af908455fedb3ea83ffc9/spec/injections.spec.js#L39-L54

and 

https://github.com/mozilla-extensions/webcompat-addon/blob/72e35293a0e4f4a0c34af908455fedb3ea83ffc9/spec/injections.spec.js#L152-L159

Just to understand what is to be removed or not.

Reply:
Yeah, good point, this test and one below it can be removed